www/src/components/plugin-searchbar-body.js

```diff
@@ -467,17 +467,16 @@ class PluginSearchBar extends Component {
 
   urlToSearch = () => {
     if (this.props.location.search) {
-      // ignore this automatically added query parameter
-      const search = this.props.location.search
-        .replace(`no-cache=1`, ``)
-        .slice(2)
+      const params = new URLSearchParams(this.props.location.search)
+      // only take into account 'search' query
+      const search = params.get(`search`)
```

Comment:
Would be great if we don't change param name (or maybe just support previous one) - so something like
```
const search = params.get(`search`) || params.get(``)
```
(dependning if we end up using `URLSearchParams` or not, see comment above)

Reply:
Guess we can use `qs` instead of `URLSearchParams` since it's also being used in other places. Supporting the old behavior might be tricky though since `qs.parse('?=gatsby')` would result in `{}`. Anw I can give it a try.

Reply:
Not ideal, but we could maybe check if string starts with `?=` and replace it with something that `qs` supports

Reply:
Would it work to use a simple regular expression to extract the text we want? Since the text is prefixed with `=` I doubt that it would cause problems for other query string users. The content of props.location.serach is still url encoded so it is safe to use `&` for tokenization. 

```javascript
const extractSearch = queryString => {
  let match = /(\?|&)=([^&]+)/.exec(queryString);
  if (match) return match[2];
  return null;
};
```

Reply:
Yep. That should work. Thank you.